docs/installation_docker.md

```diff
@@ -2,21 +2,45 @@
 
 Docker is a way to move applications including the operating system to servers.
 
-## Get a loklak Docker image
+## Get a Docker image
 
 Each of the following sections depend on a docker image with a loklak server inside.
 This section shows you how to get such an image.
-Some possibilities may be used in the sections but you can mey be able to exchenge them.
+Some possibilities may be used in the sections but you can may be able to exchange them.
 
-- pull a docker image from [dockerhub](https://hub.docker.com/r/mariobehling/loklak/):
+#### Pull a loklak docker image
+Pull a docker image from [dockerhub](https://hub.docker.com/r/mariobehling/loklak/).
 
+Type this command in your terminal:
+        ```
         docker pull mariobehling/loklak
+        ```
 
-- clone the repository and build the image from source
+#### Build a docker image on your local machine
 
-        git clone https://github.com/loklak/loklak_server.git
-        cd loklak_server
-        docker build -t mariobehling/loklak -f docker/Dockerfile .
+1. Type the following commands in terminal
+   ```
+   sudo apt-get update && sudo apt-get upgrade -y
+   sudo apt-get install docker.io
+   sudo docker build https://github.com/loklak/loklak_server.git
```

Comment:
And also you don't need `sudo` to use docker, just add yourself to the `docker` group (`usermod -aG user docker`)